ua-snap/geospatial-vector-veracity
# Add new points

This PR adds a first draft of a number of new CSVs to the CVV repo for regions relevant to circumpolar datasets to be included in relevant searchable points

- Nunavut
- Ontario
- Quebec
- Newfoundland & Labrador
- Greenland
- Iceland
- Norway
- Sweden
- Finland
- Russia
- Faroe Islands

Requirements to test: Have QGIS installed

To test
- Check that CSVs are formatted the same as previously integrated CSVs (order of columns, sig figs of lat/lons, etc...)
- Add files to GIS ecosystem alongside a base layer and check for outliers (note Nunavut and Norway files inherently contain outlier points).
- Consider whether we need to document these a source for this or whether it falls into the category of "general information".
- Check to ensure useful encoding to display Canadian Aboriginal Syllabics and Cyrillic text included in Nunavut and Russia files
- Consider scope and whether this quantity of points is going to cause problems in the 'community search' box and whether we should trim these by latitude. The Ontario and Quebec files for instance include ALL places in those provinces, but most of the southern ones are not of import to us while we do want to include northerly points. I started by including ALL points in ALL nations with any territory north of 55 degrees latitude but we can trim from there if that's a better user experience.
